tutorials/ngsi_v2/e6_timeseries_data/e6_timeseries_data_solution.py

```diff
@@ -44,49 +45,61 @@
 from tutorials.ngsi_v2.simulation_model import SimulationModel
 
 # ## Parameters
-# ToDo: Enter your context broker host and port, e.g http://localhost:1026
+# ToDo: Enter your context broker host and port, e.g. http://localhost:1026.
 CB_URL = "http://localhost:1026"
-# ToDo: Enter your IoT-Agent host and port, e.g http://localhost:4041
+# ToDo: Enter your IoT-Agent host and port, e.g. http://localhost:4041.
 IOTA_URL = "http://localhost:4041"
-# ToDo: Enter your IoT-Agent host and port, e.g http://localhost:4041
+# ToDo: Enter your IoT-Agent host and port, e.g. http://localhost:4041.
 QL_URL = "http://localhost:8668"
-# ToDo: Enter your mqtt broker url, e.g mqtt://test.mosquitto.org:1883
+# ToDo: Enter your mqtt broker url, e.g. mqtt://test.mosquitto.org:1883.
 MQTT_BROKER_URL_EXPOSED = "mqtt://localhost:1883"
-# ToDo: Enter your mqtt broker url, e.g mqtt://mosquitto:1883
+# ToDo: Enter your mqtt broker url, e.g. mqtt://mosquitto:1883.
 MQTT_BROKER_URL_INTERNAL = "mqtt://mosquitto:1883"
-# ToDo: If required enter your username and password
+# ToDo: If required, enter your username and password.
 MQTT_USER = ""
-MQTT_PW =  ""
+MQTT_PW = ""
 
+# ToDo: Change the name of your service to something unique. If you run
+#  on a shared instance this is very important in order to avoid user
+#  collisions. You will use this service through the whole tutorial.
+#  If you forget to change it, an error will be raised!
 # FIWARE-Service
 SERVICE = 'filip_tutorial'
-# FIWARE-Servicepath
-# ToDo: Change the name of your service-path to something unique. If you run
-#  on a shared instance this very important in order to avoid user
-#  collisions. You will use this service path through the whole tutorial.
-#  If you forget to change it an error will be raised!
-SERVICE_PATH = '/your_path'
-APIKEY = SERVICE_PATH.strip('/')
+# FIWARE-Service path
+SERVICE_PATH = '/'
+
+# ToDo: Change the APIKEY to something unique. This represents the "token"
+#  for IoT devices to connect (send/receive data) with the platform. In the
+#  context of MQTT, APIKEY is linked with the topic used for communication.
+APIKEY = 'your_apikey'
 UNIQUE_ID = str(uuid4())
 TOPIC_CONTROLLER = f"fiware_workshop/{UNIQUE_ID}/controller"
 print(TOPIC_CONTROLLER)
 
-# Path to read json-files from previous exercises
+# path to read json-files from previous exercises
 READ_GROUPS_FILEPATH = \
     Path("../e5_iot_thermal_zone_control_solution_groups.json")
 READ_DEVICES_FILEPATH = \
     Path("../e5_iot_thermal_zone_control_solution_devices.json")
 READ_SUBSCRIPTIONS_FILEPATH = \
     Path("../e5_iot_thermal_zone_control_solution_subscriptions.json")
 
+# opening the files
```

Comment:
Opening file same as in the previous exercises. These files will be used in this exercise.